data-science-onramp/ai-platform/trainer/sklearn_model/hptuning_config.yaml

```diff
@@ -0,0 +1,17 @@
+trainingInput:
```

Comment:
lower priority, but comments about why these are set to what they are might be helpful